### PR TITLE
Icon pack

### PR DESCRIPTION
**Pull request by henryhadlow**
All the icons.

See _icons.scss or _header.html.erb for how to use

### CODE REVIEW

**Comment by otlaitil** [comment]
Why are we not using the asset pipeline to serve the icons?

File: app/views/shared/_os.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!-- Desktop Browsers -->
+<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
+
+<!-- Android: Chrome M39 and up-->
+<link rel="manifest" href="/manifest.json">
+
+<!-- Android: Chrome M31 and up, ignored if manifest is present-->
+<meta name="mobile-web-app-capable" content="yes">
+<link rel="icon" sizes="192x192" href="/images/os-social/android/icon-192x192.png">
```
Comment (alexander-bobin) [inline app/views/shared/_os.html.erb]: Seems like this file does not exist

**Comment by alexander-bobin** [comment]
Maybe we could include our images in the asset pipeline as part of another story?

**Comment by alexander-bobin** [comment]
LGTM

**Pull request merged**